packages/components/src/Enumeration/Header/headerSelected.snapshot.test.js

```diff
@@ -38,7 +38,13 @@ describe('Header', () => {
 	it('should render header in selected state without trash icon in case of single selection)', () => {
 		// given
 		const props = {
-			headerSelected: [],
+			headerSelected: [{
```

Comment:
This test is meaningless now. `should render header in selected state without trash icon in case of single selection)`
Remove it